### ISSUE TITLE
The theme of the whole site 

### ISSUE THREAD

**Issue by nuwrss** [WebAhead7/memory-game#18]
you have to make a unified theme for the whole website .

**Comment by halaassaly** [comment]
You are totally right, because of the lake of time we didn't design it as we should. But we will try to do so in the future projects. 👍

**Comment by nuwrss** [comment]
We are just in the start of our journey ;-)
